### PR TITLE
chore: propose PR checklist template

### PR DESCRIPTION
**Pull request by degliwe**

### CODE REVIEW

**Comment by emeryro** [comment]
addition proposals:
- [ ] I have added the components directly used in the template as dependencies
- [ ] my variables naming follow the guidelines (snake case for twig, camel case for javascript)
- [ ] I have provided documentation (for the "notes" tab)

**Comment by yhuard** [comment]
- `camel case for javascript`: this is already checked by the linter, we don't need to add it to the checklist
- `I have put the original css as dependency`: not only the "CSS" (prefer "vanilla"), but also the "specs" package
- shall we add a bullet about the GH project or do we abandon it?

**Comment by yhuard** [comment]
one more thing: 
> My component is listed in `@ecl-twig/ec-components`'s dependencies

**Review by yhuard — CHANGES_REQUESTED** [review]
> My component is listed in `@ecl-twig/ec-components`'s dependencies

And `pr: review needed` is only valid if the PR actually needs a review (i.e. not WIP). The rule could be more generic ("My PR has a `pr: *` label")

**Comment by emeryro** [comment]
-  I have added the components directly used in the twig file (with `include`) as dependencies

Also for vanilla and specs we should say `devDependencies` instead of `dependencies` (will prevent back and forth)

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+Please drop a few lines about the PR: what it does, how to test it, etc.
+
+## QA Checklist
+
+In order to ensure a safe and quick review, please check that your PR follow those guidelines:
+
+* [ ] I have put the vanilla component as `devDependencies`
+* [ ] I have put the specs package as `devDependencies`
+* [ ] I have added the components directly used in the twig file (with `include`) as dependencies
```
Comment (yhuard) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+* [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
+* [ ] My variables naming follow the guidelines (snake case for twig)
+* [ ] I have provided tests
+* [ ] I have provided documentation (for the "notes" tab)
+* [ ] My local `yarn.lock` was commited
```
Comment (yhuard) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
* [ ] If my local `yarn.lock` contains changes, I have committed it
```

**Review by yhuard — CHANGES_REQUESTED** [review]

**Review by yhuard — APPROVED** [review]

**Pull request merged**